.gitignore

```diff
@@ -46,6 +46,10 @@ packages/
 /*.XML
 *.nupkg
 
+# AppSettings
+appsettings.development.json
```

Comment:
Don't these new files need to be added to the respective csprojs, and copied to the output folder?  Similar to
`    <Content Update="appsettings.json">
      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
    </Content>
`
defined in the test harness csproj.  Maybe we can make this work with a **.json filter and just grab them all?  I ran into issues when trying to run the postman tests, where the json file didn't get updated in the bin/debug folder.

Reply:
added **.json to the .csproj